srp_daemon/start_on_all_ports

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
```

Comment:
In the description, please consider changing "tart_on_all_ports" into "start_on_all_ports" and "compatability" into "compatibility".